www/reviews.php

```diff
@@ -450,7 +458,10 @@ function showReview($db, $gameid, $rec, $specialNames, $optionFlags = 0)
             . "return false;\">Yes</a> &nbsp; "
             . "<a href=\"needjs\""
             . "onclick=\"javascript:sendReviewVote('$reviewid', 'N');"
-            . "return false;\">No</a> &nbsp; ";
+            . "return false;\">No</a> &nbsp; "
+            . "<a href=\"needjs\""
+            . "onclick=\"javascript:sendReviewVote('$reviewid', 'R');"
+            . "return false;\">Reset</a> &nbsp; ";
```

Comment:
```suggestion
            . "return false;\">Remove vote</a> &nbsp; ";
```

Seems like on https://github.com/iftechfoundation/ifdb-suggestion-tracker/issues/195 folks liked "Remove vote"